Bottom: 4f274b70cbd78cebf284af940b7b97bcac26361c
Top:    44afd53886ae7b4e89c45e06646b61691e5b2918
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 21:01:21 +0000

Refresh of island-removal-fixes

---

diff --git a/src/buffer.c b/src/buffer.c
index 050648f..8cf8096 100644
--- a/src/buffer.c
+++ b/src/buffer.c
@@ -389,7 +389,6 @@ MovePourToBuffer (LayerTypePtr Layer, PourTypePtr Pour)
   PourTypePtr pour;
 
   r_delete_entry (Layer->pour_tree, (BoxTypePtr) Pour);
-  RestoreToPours (Source, POUR_TYPE, Layer, Pour);
   lay = &Dest->Layer[GetLayerNumber (Source, Layer)];
   pour = GetPourMemory (lay);
   *pour = *Pour;
diff --git a/src/move.c b/src/move.c
index 01003fb..f9a80bd 100644
--- a/src/move.c
+++ b/src/move.c
@@ -367,7 +367,6 @@ MovePour (LayerTypePtr Layer, PourTypePtr Pour)
       ErasePour (Pour);
     }
   r_delete_entry (Layer->pour_tree, (BoxType *) Pour);
-  RestoreToPours (PCB->Data, POUR_TYPE, Layer, Pour);
   MovePourLowLevel (Pour, DeltaX, DeltaY);
   r_insert_entry (Layer->pour_tree, (BoxType *) Pour, 0);
   InitPourClip (PCB->Data, Layer, Pour);
@@ -433,7 +432,6 @@ MovePourPoint (LayerTypePtr Layer, PourTypePtr Pour, PointTypePtr Point)
       ErasePour (Pour);
     }
   r_delete_entry (Layer->pour_tree, (BoxType *) Pour);
-  RestoreToPours (PCB->Data, POUR_TYPE, Layer, Pour);
   MOVE (Point->X, Point->Y, DeltaX, DeltaY);
   SetPourBoundingBox (Pour);
   r_insert_entry (Layer->pour_tree, (BoxType *) Pour, 0);
